tests/behat/bootstrap/FeatureContext.php

```diff
@@ -522,4 +522,43 @@ public function wysiwygFillField(string $field, string $value): void {
         ");
   }
 
+  /**
+   * Assert element with wildcard pattern exists.
+   *
+   * An element with selector and attribute with a value exists,
+   * matching a wildcard pattern.
+   *
+   * @Then I( should) see the :selector element with the :attribute attribute like :pattern
+   */
+  public function elementAssertAttributeLikePattern(string $selector, string $attribute, string $pattern): void {
```

Comment:
This does not need to check for the wildcard - it checks if the attribute contains a string.

Please update to 
```
@Then I( should) see the :selector element with a(n) :attribute attribute containing :value
```

The method then will become `elementAssertAttributeContains`